src/fiskaltrust.Launcher/Services/HostingService.cs

```diff
@@ -239,7 +246,23 @@ private BasicHttpBinding CreateBasicHttpBinding(BasicHttpSecurityMode securityMo
                 _logger.LogWarning($"{nameof(_launcherConfiguration.UseHttpSysBinding)} is not supported for grpc.");
             }
 
-            builder.WebHost.BindKestrel(uri, listenOptions => ConfigureTls(listenOptions), false, HttpProtocols.Http2);
+            // Support for domain sockets/namespots
+            if (_launcherConfiguration.DomainSocket.Enabled)
+            {
+                builder.WebHost.UseKestrel(options =>
+                {
+                    options.ListenUnixSocket(_launcherConfiguration.DomainSocket.Path!, listenOptions =>
+                    {
+                        listenOptions.Protocols = HttpProtocols.Http2;
+                        ConfigureTls(listenOptions);
+                    });
+                });
+            }
+            else
+            {
```

Comment:
The hosting service is only used to host the urls for `Queues` `SCU`s and Helpers.

We don't want to change anything in the hosting for them so we don't need to touch the hosting service.
```suggestion
```